src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetDraftApprentice/GetDraftApprenticeQueryValidatorTests.cs

```diff
@@ -0,0 +1,35 @@
+using FluentValidation.Results;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetDraftApprentice;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetProvider;
+
+namespace SFA.DAS.CommitmentsV2.UnitTests.Application.Queries.GetDraftApprentice
+{
+    [TestFixture]
+    [Parallelizable]
+    public class GetDraftApprenticeValidatorTests
+    {
+        [TestCase(-1, -1, false)]
+        [TestCase( 0, -1, false)]
+        [TestCase(-1,  0, false)]
+        [TestCase( 0,  0, false)]
+
+        [TestCase( 1,  0, false)]
+        [TestCase( 0,  1, false)]
+        [TestCase( 1,  1, true)]
+
+        public void Validate_WhenValidating_ThenShouldValidate(long cohortId, long draftApprenticeshipId,  bool expectedValue)
```

Comment:
Can this be renamed? It looks like it's testing that both cohort id and draft apprenticeship are positive ints, but the name is unclear.

Reply:
Yep. That test is a bit weird. Not sure what I was thinking.